### PR TITLE
fix(dedicatedcloud): change the way to fetch the catalog

### PR DESCRIPTION
**Pull request by antleblanc**
| Question         | Answer
| ---------------- | ---
| Branch?          | `release/2020-w25`
| Bug fix?         | no
| New feature?     | no
| Breaking change? | no
| Tickets          | no
| License          | BSD 3-Clause

### 🐛 Bug Fixes

d884fe6 - fix(dedicatedcloud): change the way to fetch the catalog

### 🏠 Internal

No quality check required.

### CODE REVIEW

File: packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js [hunk 1]
```diff
@@ -117,8 +118,13 @@ export default class BackupService {
   }
 
   getCatalog(ovhSubsidiary) {
-    return this.catalogApi.get({ catalogName: 'privateCloud', ovhSubsidiary })
-      .$promise;
+    return this.$http
+      .get(`2api/sws/dedicatedcloud/catalog`, {
```
Comment (frenautvh) [inline packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js:122]:
```js
get('/sms/dedicatedcloud/catalog', {
  serviceType: 'aapi',
  params: { ... },
})

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js:122]: @frenautvh I can confirm that the URL is the correct one.

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js:122]: You mean passing the `serviceType` parameter?

Comment (frenautvh) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js:122]: yes i though we had to set the serviceType but in core, an interceptor is setting it automatically if your url starts with 2api so i guess it's fine :)
see : https://github.com/ovh/manager/blob/master/packages/manager/modules/core/src/index.js#L176

Comment (antleblanc) [reply packages/manager/apps/dedicated/client/app/dedicatedCloud/datacenter/backup/backup.service.js]: Thanks for pointing out this.

I've added the `serviceType` parameter and remove extra template strings.

**Pull request merged**